src/data/get_flight_stats.py

```diff
@@ -27,18 +27,18 @@ class FlightStats:
     _flight_stats_df: pd.DataFrame = field(init=False)
 
     def __post_init__(self, flights: pd.DataFrame) -> None:
-        self._flight_stats_df = flights.map(self.count_flights)
+        flight_stats_list = list(map(self.count_flights, flights['flights'], flights['icao']))
```

Comment:
Is `icao` always present in the production data?

Reply:
Yeah it is